src/SFA.DAS.RoatpGateway.Web/Controllers/RoatpGatewayController.cs

```diff
@@ -31,10 +31,16 @@ public class RoatpGatewayController : RoatpGatewayControllerBase<RoatpGatewayCon
         public async Task<IActionResult> NewApplications(int page = 1)
         {
             var applications = await _applyApiClient.GetNewGatewayApplications();
+            var counts = await _applyApiClient.GetApplicationCounts();
 
             var paginatedApplications = new PaginatedList<RoatpApplicationSummaryItem>(applications, applications.Count, page, int.MaxValue);
 
-            var viewModel = new RoatpGatewayDashboardViewModel { Applications = paginatedApplications };
+            var viewModel = new RoatpGatewayDashboardViewModel
+            {
+                Applications = paginatedApplications,
+                ApplicationCounts = counts,
+                SelectedTab = "NewApplications"
```

Comment:
Personally, I'd use `nameof(NewApplications)` in case it ever changes

Reply:
Agreed, done